python/rmm/_cuda/gpu.pyx

```diff
@@ -275,10 +70,34 @@ def driverGetVersion():
     """
     cdef int version
     cdef cudaError_t status = cudaDriverGetVersion(&version)
-    if status != 0:
+    if status != cudaError.cudaSuccess:
         raise CUDARuntimeError(status)
     return version
 
+def get_current_device():
+    """
+    Get the current CUDA device
+    """
+    cdef int current_device
+    cdef cudaError_t status = cudaGetDevice(&current_device)
+    if status != cudaError.cudaSuccess:
+        raise CUDARuntimeError(status)
+    return current_device
+
+
+def set_current_device(int device):
+    """
+    Set the current CUDA device
+    Parameters
+    ----------
+    device : int
+        The ID of the device to set as current
+    """
+    cdef cudaError_t status = cudaSetDevice(device)
+
+    if status != cudaError.cudaSuccess:
+        raise CUDARuntimeError(status)
```

Comment:
Can we make these more analogous to `cudaSetDevice` / `cudaGetDevice`? i.e. something like `setDevice` / `getDevice` for now?